Dockerfile.thanos-ci

```diff
@@ -1,5 +1,5 @@
 # Available from https://hub.docker.com/r/circleci/golang/
-FROM circleci/golang:1.13.6
+FROM circleci/golang:1.13.6-node
```

Comment:
Did not find which Node.js version runs in this image (Could possibly lead to a build problems later). Better to fix it `package.json` to LTS if possible https://docs.npmjs.com/files/package.json#engines